scripts/linters/css_linter.py

```diff
@@ -105,20 +95,16 @@ def _lint_css_files(self):
                 '         or node-stylelint and its dependencies.')
             sys.exit(1)
         files_to_lint = self.all_filepaths
-        start_time = time.time()
         num_files_with_errors = 0
-        summary_messages = []
+        failed = False
+        error_messages = []
```

Comment:
Could we name error_messages to may be `stripped_error_messages`? Then it would be easy to differentiate between full vs stripped. Though, there could be some other better name.